build.sbt

```
@@ -54,6 +54,48 @@ commands += Command.command("save-expect") { s =>
 commands += Command.command("save-manifest") { s =>
   "testsJVM/test:runMain scala.meta.tests.semanticdb.SaveManifestTest" :: s
 }
+
+commands += Command.command("gh-actions-release") { s =>
+  val isTag = Option(System.getenv("GITHUB_REF"))
+    .exists(_.startsWith("refs/tags"))
+
+  val tagRegex = "^v\\d+\\.\\d+\\.\\d+.*".r
```

```suggestion
  val tagRegex = "^v\\d+\\.\\d+\\.\\d+$".r
```
Is the last one needed ?